\section{CONCLUSIONS}
To conclude, several simulations of 2 Mb chromatinic regions containing the ANPEP locus were performed. The segment was considered as composed by beads with fixed dimension (chapter \ref{chap: the model description}), which were assigned to one of the state presented in the \textit{ChromHMM} results (chapters and \ref{intro: chromhmm} and \ref{methods: finding enriched states}) whose input were ATAC-seq and CTCF Chip-Seq data (chapter \ref{methods: data used}). After the tuning of the parameters associated to the interaction potentials generated between beads of the same type, very interesting correlation coefficients between the simulated matrices and the true experimental matrices were found. %#TODO add considerations about hte matrices similarities
The maps were compared making use of the SCC coefficient, however, another possible way make the comparison would be to compute the Spearman correlation coefficient. Ideally, it would be interesting to see if there are cases where the two metrics produce different results of SCC and Spierman correlation coefficient, and to understand which of them is better in what cases.
A potential.
As a future perspective, it could be considered the extension of the analysis towards new cell-types and/or new \textit{loci}. In particular, we would be interested in investigating the MYC, SOX9, ITG45, MSX2, NT5E genes, and the GM12878 cell-type. Also, the tuning process for the parameters could be improved and automated better.
To allow a better comparison with the already present models, the results obtained with the model could be compared to those resulting from other very interesing simulation softwares, such as \textit{Origami} and \textit{Hip-Hop}
\cite{bucklePolymerSimulationsHeteromorphic2018,tanCelltypespecificPrediction3D2023}.




