\subsection{\textit{ChromHMM} allows the sequential characterization of chromatin states} \label{intro: chromhmm}
\textit{ChromHMM}
\cite{chilledhousevibesLearningChromatinStates2015}
is a tool which helps in the annotation of genomic DNA by using epigenomic information
\cite{ernstChromatinstateDiscoveryGenome2017}.
It learns chromatin states signatures by using a multivariate hidden Markov model: in each genomic position (segment), it returns the most probable chromatin state and gives other useful information
\cite{chilledhousevibesLearningChromatinStates2015,ernstChromatinstateDiscoveryGenome2017}. 

The package works through two functions in particular, which are the following
\cite{ernstChromatinstateDiscoveryGenome2017}:
\begin{enumerate}
    \item \textbf{\textit{BinarizeBam}}: it converts a set of \textit{.bam} files of aligned reads into binarized data files in a specified output directory. The produced data can be used as input for the \textit{LearnModel} function. When using this command, it has to be specified the segment size, which is set by default to be equal to 200 bps.
    \item  \textbf{\textit{LearnModel}}: it takes a set of binarized data files, learns chromatin state models, and by default produces data reporting the emission/transition parameters of the states, the abundance of the states at the TSSs (Transcriptional Starting Sites), at the TESs (Transcriptional Ending sites), and other relevant portions of the genome (CPG islands, exons, genes). Additionally, a webpage is generated with links to all the files and images created.
\end{enumerate}

\noindent The results obtained from \textit{ChromHMM} are shown in chapter \ref{chap: ChromHMM results}.
